lesson_27/kimberleeh/src/pages/Home/components/Program/Program.tsx

```diff
@@ -0,0 +1,14 @@
+import './Program.scss';
+import React, {Key, PropsWithChildren} from 'react';
+
+export interface ProgramProps extends PropsWithChildren {
+  id: Key | null | undefined;
+  title: string;
+}
+
+export const Program: React.FC<ProgramProps> = ({title, children}: ProgramProps) => {
+    return (<li className="program">
+        <h3>{title}</h3>
+        {children}
```

Comment:
Although you could use `children`, that's probably not what you want to do. You should use the `description` property I recommend above.